packages/fondue/src/components/Tooltip/Tooltip.tsx

```diff
@@ -36,22 +36,21 @@ const formatTooltipText = (text: string) => {
     return text.split(lineBreakRegex).join('\n');
 };
 
-type TimeoutRef = {
-    current: NodeJS.Timeout | null;
-};
-
-const handleTimeout = (callback: () => void, delay: number, timeoutRef: TimeoutRef) => {
-    if (timeoutRef.current) {
-        clearTimeout(timeoutRef.current);
+let timeoutId: NodeJS.Timeout | undefined;
```

Comment:
Also 
```suggestion
let timeoutId: number | undefined;
```
and you need to use `window.setTimeout` instead of just `timeout`

Reply:
Updated.